featuretools/entityset/entityset.py

```diff
@@ -1256,26 +1249,12 @@ def _add_multigenerational_link_vars(self, frames, start_entity_id,
                                                              left_on=r.child_variable.id,
                                                              right_on=r.child_variable.id)
 
-    @classmethod
-    def _load_dummy_entity_data_and_variable_types(cls, metadata):
-        variable_types = {}
-        defaults = []
-        columns = []
-        variable_names = {}
-        for elt in dir(vtypes):
-            try:
-                cls = getattr(vtypes, elt)
-                if issubclass(cls, vtypes.Variable):
-                    variable_names[cls._dtype_repr] = cls
-            except TypeError:
-                pass
-        for vid, vmetadata in metadata['variables'].items():
-            if vmetadata['dtype_repr']:
-                vtype = variable_names.get(vmetadata['dtype_repr'], vtypes.Variable)
-                variable_types[vid] = vtype
-                defaults.append(vtypes.DEFAULT_DTYPE_VALUES[vtype._default_pandas_dtype])
-            else:
-                defaults.append(vtypes.DEFAULT_DTYPE_VALUES[object])
-            columns.append(vid)
-        df = pd.DataFrame({c: [d] for c, d in zip(columns, defaults)}).head(0)
-        return df, variable_types
+
+def read_entityset(path, **kwargs):
+    '''Read entityset from disk.
+
+        Args:
+            path (str): Directory on disk to read `data_description.json`.
+            kwargs (keywords): Additional keyword arguments to pass as keyword arguments to the underlying deserialization method.
+    '''
+    return EntitySet.from_data_description(deserialize.read_data_description(path), **kwargs)
```

Comment:
let's make this two lines

```
data_description = deserialize.read_data_description(path)
return EntitySet.from_data_description(data_description, **kwargs)
```

Reply:
Change applied in [this commit.](https://github.com/Featuretools/featuretools/pull/361/commits/6d08df367f468758a3efbc2001cf7094c02f30a7#diff-8ca005f60a194e43663c2a5128774ecdR1260)